.eslintrc

```diff
@@ -1,3 +1,7 @@
 {
-    "extends": "react-app"
+    "extends": "react-app",
```

Comment:
Thanks!

I'm not sure about presets for react. There is often the 'extends' of other configs, just as `eslint-config-airbnb` I've not tried to see if that works.